pkg/p2p/libp2p/internal/handshake/handshake_test.go

```diff
@@ -63,15 +63,15 @@ func (s *StreamMock) Close() error {
 func TestHandshake(t *testing.T) {
 	logger := logging.New(ioutil.Discard, 0)
 	info := Info{
-		Address:   "node1",
+		Address:   []byte("node1"),
```

Comment:
Use swarm.Address not []byte as type, and construct it from actual byteslice or use swarm.NewAddress if it is constructed from hex-encoded string.